src/content/self-hosted/administration/configuration.mdx

```diff
@@ -461,7 +461,16 @@ The daemonset automatically configures every node of your cluster to work better
 - `extraEnv`: Environment variables to be set on the daemonset containers.
 - `labels`: Labels to add to the daemonset pods.
 - `image`: Container image used by the daemonset pods.
-- `initContainers.privateCA.image`: InitContainer image used by the daemon pods. Defaults to `busybox`
+
+The daemonset does the following tasks on each node:
+
+- [Overrides](self-hosted/administration/configuration.mdx#overrideregistryresolution) the Okteto Registry hostname resolution to use internal IPs.
+- [Overrides](self-hosted/administration/configuration.mdx#overridefilewatchers) the default kernel values for file watchers on every node.
+- Configures the kubelet with registry credentials for [private registries](self-hosted/administration/configuration.mdx#privateregistry).
+- Installs your CA if `wildcardCertificate.privateCA` is enabled.
+- Installs a CA if using self-signed certificates (`wildcardCertificate.create: true`).
+
+You can restrict the nodes where the daemonset is deployed using the `tolerations.devPool` field.
```

Comment:
Can you explain how? Maybe include a sample? I think this configuration is not trivial to get right.